README.adoc

```diff
@@ -392,6 +392,129 @@ endif::[]
 
 You can externalize the configuration of more than just the port numbers. To learn more about Open Liberty server configuration, check out the https://openliberty.io/docs/latest/reference/config/server-configuration-overview.html[Server Configuration Overview^] docs. 
 
+== Optimizing the image size
+
+As previously mentioned, the parent image used in each `Dockerfile` contains the `full` tag which includes all of the Liberty features. This is recommended for development but when deploying to production it's recommended to use a parent image with the `kernel-slim` tag. The `kernel-slim` tag provides a bare minimum server with the ability to add the features required by the application.
+
+[role="code_command hotspot file=0", subs="quotes"]
+----
+#Replace the `Dockerfile` for the inventory service.#
+`inventory/Dockerfile`
+----
+
+inventory/Dockerfile
+[source, Text, linenums, indent=0, role="code_column"]
+----
+include::finish/inventory/Dockerfile[]
+----
+
+Replace the parent image with [hotspot=kernel-slim file=0]`icr.io/appcafe/open-liberty:kernel-slim-java11-openj9-ubi` at the top of your Dockerfile. This image contains the `kernel-slim` tag that's recommended when deploying to production.
+
+Place [hotspot=features file=0]`RUN features.sh` command after the `COPY` command that the configuration file is copied to the `/config/` destination directory. The `features.sh` script adds the Liberty features that your application is required to operate.
```

Comment:
Please rephrase this sentence. it is confusing.
Place [hotspot=features file=0]`RUN features.sh` command after the `COPY` command that the configuration file is copied to the `/config/` destination directory.